File: 09challenge/main.go
package main

import (
    "fmt"
)

func main() {
    // Declare a variable to store the amount of money in the person's wallet (in Brazilian Real)
    var money_in_wallet float64

    // Get the amount of money in the person's wallet (in Brazilian Real)
    fmt.Print("Enter the amount of money in your wallet (in Brazilian Real): ")
    fmt.Scan(&money_in_wallet)

    // Declare a variable to store the exchange rate
    exchange_rate := 3.45

    // Calculate the amount of money in dollars
    money_in_dollars := money_in_wallet / exchange_rate

    // Print the amount of money in dollars
    fmt.Printf("You can buy US$%.2f with the money in your wallet.\n", money_in_dollars)
}

File: 09challenge/main.cpp
#include <iostream>
using namespace std;

int main() {
    // Declare a variable to store the amount of money in the person's wallet (in Brazilian Real)
    float money_in_wallet;

    // Get the amount of money in the person's wallet (in Brazilian Real)
    cout << "Enter the amount of money in your wallet (in Brazilian Real): ";
    cin >> money_in_wallet;

    // Declare a variable to store the exchange rate
    float exchange_rate = 3.45;

    // Calculate the amount of money in dollars
    float money_in_dollars = money_in_wallet / exchange_rate;

    // Print the amount of money in dollars
    cout << "You can buy US$" << money_in_dollars << " with the money in your wallet." << endl;

    return 0;
}
